src/main/kotlin/day11/DefaultFileOperations.kt

```
@@ -0,0 +1,23 @@
+import FileOperations
+import java.io.File
+import java.io.FileNotFoundException
+
+class DefaultFileOperations : FileOperations {
+    override fun copy(sourcePath: String, targetPath: String) {
+        try {
+            val originalFile = File(sourcePath)
+            val copyFile = File(targetPath)
+            copyFile.writeText(originalFile.readText())
+
+        } catch (e: FileNotFoundException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)